src/gui.rs

```diff
@@ -93,7 +92,11 @@ impl GUI {
     pub fn post_render(&mut self) {
         self.staging_belt.recall();
     }
-
+    
+    pub fn change_state(&mut self, message: Message) {
```

Comment:
Why not just name this `queue_message` also?

Reply:
That wouldn't indicate what the message is for.